src/Controller/UserAccount/ParticipationController.php

```diff
@@ -73,10 +73,10 @@ public function userParticipation(Request $request): Response
         ]);
     }
 
-    #[Route(path: '/user/future-participations', name: 'future_participations')]
+    #[Route(path: '/user/future-participations', name: 'accepted_participations')]
```

Comment:
This move is weird :/ 

There is an inconsistency between the 
* path & methd name
* VS name & template name

Reply:
It seems I forgot to change the path name, however the template name was already `accepted_participation`.
I made this change for consistency because we are naming things after our workflow constants pretty much everywhere but here.

It was also error prone as well as a bit misleading so I'll just double check I didn't forget any other `futre_xxx` stuff